src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
Consider using a private constructor in the enum to prevent the instantiation of the enum class.